Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Done/DefaultDoneFormButtonStyle.swift

```diff
@@ -12,7 +14,9 @@ struct DefaultDoneFormButtonStyle: CKOButtonStyle {
     var height: Double
     var width: Double
     
-    init( isEnabled: Bool = true,
+    init( textColor: UIColor = .clear,
+          isHidden: Bool = false,
+          isEnabled: Bool = true,
```

Comment:
Same as `DefaultCancelButtonFormStyle` around sorting properties and initialiser in a more logical way.

Reply:
please add suggestion here using <Cmd+g>

Reply:
I am not familiar with the component completely, I think its the sort of thing that takes practice and best done on first hand. I wouldn't be able to interpret it properly from an out of context code review, so if you say this is appropriate, that is fine with me